README.md

```diff
@@ -1,2 +1,60 @@
-# TicTacToeRubyRails
-Tic Tac Toe in Ruby on Rails
+# TicTacToeRails::API
+
+[![Build Status](https://travis-ci.com/mrfishball/TicTacToeRubyRails.svg?branch=master)](https://travis-ci.com/mrfishball/TicTacToeRubyRails)
+
+**TicTacToeRails::API** is alive [@Heroku](https://live-ttt-rubyrails-api.herokuapp.com)!
+
+It is used to implement your very own Tic-Tac-Toe game on virtually any platform of your choice.
+
+### What is an API app?
+
+**TicTacToeRails::API** provides a programmatically accessible API alongside their web application.
+For example, GitHub provides [an API](http://developer.github.com) that you can use from your own custom clients.
+
+With the advent of client-side frameworks, more developers are using Rails to build a backend that is shared between their web application and other native applications.
+
+For example, Twitter uses its [public API](https://dev.twitter.com) in its web application, which is built as a static site that consumes JSON resources.
+
+### Documentation
+
+WIP...
```

Comment:
WIP?

Reply:
should have been work in progress. will change that